django/http/request.py

```diff
@@ -89,14 +90,47 @@ def headers(self):
 
     @cached_property
     def accepted_types(self):
-        """Return a list of MediaType instances."""
-        return parse_accept_header(self.headers.get("Accept", "*/*"))
+        """Return a list of MediaType instances, in order of preference."""
+        header_value = self.headers.get("Accept", "*/*")
+        return sorted(
+            (MediaType(token) for token in header_value.split(",") if token.strip()),
+            key=operator.attrgetter("quality", "specificity"),
+            reverse=True,
+        )
```

Comment:
🤔 Changing this to be _ordered_ by preference/precedence instead a reflection of what was provided in the header is fine by my guess (😆), but it _is_ a change. I'd not expect anyone to just be plucking out `[0]` and hoping for the best, but if someone is, this could be confusing. 
I'd suggest it ought to be mentioned specifically in the changelog, when it comes to that.

Reply:
As long as the release notes clearly call out the behaviour, it's probably OK.

Reply:
Sure, let's add this to the `Miscellaneous` section in 5.2